system/View/Traits/DecoratesViews.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace CodeIgniter\View\Traits;
+
+use CodeIgniter\View\Exceptions\ViewException;
+use CodeIgniter\View\Interfaces\ViewDecorator;
+
+use function config;
+
+trait DecoratesViews
```

Comment:
I really like the Symfony-esque trait naming scheme... but this would be our first. All our other traits have boring names, à la `ViewDecoratingTrait`. I'm not opposed, just pointing it out. 

Regardless, let's make a version 5 card entry for "Trait Naming Schemes"!